Fluid.MvcViewEngine/FluidMvcViewOptions.cs

```diff
@@ -1,9 +1,16 @@
 ﻿using Fluid.ViewEngine;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using System.Threading.Tasks;
 
 namespace Fluid.MvcViewEngine
 {
     public class FluidMvcViewOptions : FluidViewEngineOptions
     {
-        // Placeholder for future options specific to the MVC view engine
+        public delegate ValueTask RenderingMvcViewDelegate(string path, ViewContext viewContext, TemplateContext context);
+
+        /// <summary>
+        /// Gets or sets the delegate to execute when a view is rendered.
+        /// </summary>
+        public new RenderingMvcViewDelegate RenderingViewAsync { get; set; }
```

Comment:
the `new` sounds like trouble, should it be separate property?

Reply:
What kind of trouble do you see? There is no point in exposing the base one in this case since the delegate has more arguments. But I don't want it to behave randomly. I expect this property to just replace the base one. Or that the base one is null when invoked as the base type.

Reply:
Checked the documentation and the based property will remain null, unless access directly by casting. I think that's fine in this context.

Reply:
Using `new` to hide members is usually a bit of code smell as behavior bound to type used to access the thing, that is all.